runtime/rasdump/TextFileStream.cpp

```diff
@@ -51,10 +52,10 @@ void
 TextFileStream::open(const char* fileName, bool cacheWrites)
 {
 	PORT_ACCESS_FROM_PORT(_PortLibrary);
-	if (NULL != strstr(fileName, "/STDOUT/")) {
-		_FileHandle = 1; 
-	} else if (NULL != strstr(fileName, "/STDERR/")) {
-		_FileHandle = 2;
+	if (NULL != strstr(fileName, J9PRINT_TO_STDOUT)) {
```

Comment:
The reason why we use strstr is because `fileName` actually includes the path of the file (Eg: /root/build/..../FILENAME. If we used a regular strncmp, it would return false. I think the code to manually parse for the filename would be too tedious, and I don't think that there's any implemented function to get the direct file name (without the path).

Also @keithc-ca, the reason why we used slashes in the option is because it is illegal to name a file with slashes, and when we use strstr, there wont be a file with the substring `/STDOUT/`.

If there is a better solution, please let me know!

Reply:
We _want_ it to treat a path like `.../STDOUT/whatever/...` as it always has; that is why `strstr` is inappropriate.

We need to use something like `strcasecmp` for a case insensitive comparison.

Reply:
Gotcha, is there a way to grab the current directory path in openJ9? I will be needing that when using `strcasecmp` since the filaname includes the path to the file (and for strcasecmp to work, we need to compare the file name to `path+"STDOUT"`

Reply:
I'm sure there is, but that doesn't factor into the test we want; we only want to look at the `file=xxx` part of the command-line argument.